web-ng/src/app/components/map/map.component.html

```diff
@@ -17,7 +17,8 @@
 <google-map
   height="100%"
   width="100%"
-  [options]="mapOptions">
+  [options]="mapOptions"
+  (mapClick)="onFeatureClick('')">
```

Comment:
Can we abstract this action a little better, eg by adding a no-arg method `onFeatureBlur()`?

Reply:
Done.